### PR TITLE
VID/PID Request

### PR DESCRIPTION
**Pull request by SandeepanSengupta**
Request for VID/PID for open source design (OSHW)

### CODE REVIEW

File: 1209/ABCD/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: CodeBridge
+owner: Intelectron India
+license: OSHW
```
Comment (Arachnid) [inline 1209/ABCD/index.md]: OSHW isn't a license; please specify a specific license that your firmware and hardware designs are under, and include the LICENSE file for that license in your code repository.

File: 1209/ABCD/index.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+site: https://www.intelectron.in
+source: https://github.com/sandeepansengupta/Generic/tree/master/CodeBridge/CAD/Eagle
```
Comment (Arachnid) [inline 1209/ABCD/index.md]: Please also include OSS-licensed source code for your product in your source repository.

File: 1209/ABCD/index.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: CodeBridge
+owner: Intelectron India
```
Comment (Arachnid) [inline 1209/ABCD/index.md]: Just for safety, would you mind removing the space from your org directory name?

**Comment by SandeepanSengupta** [comment]
Sorry for late reply. I must admit, I'm a noob when it comes to github, can
you please elaborate. I've already included Creative Common License
agreement text to the directory. I'll be happy to comply fix things as
needed.
Regards,
Sandeepan Sengupta

**Comment by SandeepanSengupta** [comment]
T've changed org directory name from "Intelectron India" to "intelectron.in"
Is it okey ?

File: 1209/ABCD/Index.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: CodeBridge
+owner: Intelectron India
```
Comment (peternewman) [inline 1209/ABCD/Index.md]: This needs updating to intelectron.in

File: 1209/9999/Index.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: PrayogEngine
+owner: Intelectron India
```
Comment (peternewman) [inline 1209/9999/Index.md]: This needs updating to intelectron.in

File: 1209/ABCD/Index.md (hunk 4 above)
Comment (SandeepanSengupta) [inline 1209/ABCD/Index.md]: Owner information is updated to intelectron.in (from Intelectron India) as suggested

File: 1209/9999/Index.md (hunk 5 above)
Comment (SandeepanSengupta) [inline 1209/9999/Index.md]: Owner information is updated to intelectron.in (from Intelectron India) as suggested

**Comment by SandeepanSengupta** [comment]
Pull Request

**Pull request merged**